### ISSUE TITLE
Use of nested \sameword-commands

### ISSUE THREAD

**Issue by stenskjaer** [maieul/ledmac#292]
If you have a passage with nested use of the `\sameword`-command, the disambiguation seems to go awry.

I am unsure if this is the correct way to use the `[inlemma]`-command in this situation:
```latex
\documentclass{article}
\usepackage{eledmac}

\begin{document}
\beginnumbering
\pstart 
\edtext{in \edtext{\sameword{parte}}{\Bfootnote{partibus \textbf{A}.}} autem 
oculi compatiuntur, et ideo in \sameword[inlemma]{parte} corporis in mulieribus}{\lemma{in
\sameword{parte} -- mulieribus}\Bfootnote{\emph{om.} \textbf{E}.}}, sicut et 
ceterae partes corporis.
\pend
\endnumbering

\end{document}
```

This results in the following apparatus:

> 1 in parte999 – mulieribus ] om. E. 
> 1 parte999 ] partibus A.

**Comment by maieul** [comment]
I am looking on it.

**Comment by maieul** [comment]
the problem is complex. I have worked yesterday, but my solution was not good. I will try again tomorrow.

**Comment by maieul** [comment]
I have made one half of the work...it needs 6 hours...

Next for tomorrow or later

**Comment by maieul** [comment]
now, it works for nested commands, but not when \lemma are called.

**Comment by maieul** [comment]
I have found the way. Will integrate it tomorrow.

**Comment by maieul** [comment]
Ok. Now you can test the branch issue292. I will write the handbook tomorrow.

So your MWE becomes

`````
\pstart 
\edtext{in \edtext{\sameword[1]{parte}}{\Bfootnote{partibus \textbf{A}.}} autem 
oculi compatiuntur, et ideo in \sameword{parte} corporis in mulieribus}{\lemma{in
\sameword{parte} -- mulieribus}\Bfootnote{\emph{om.} \textbf{E}.}}, sicut et 
ceterae partes corporis.
\pend
``````

The `[inlemma]` is abandonned, because the `\lemma` is not the same in many level. So we replace it by a comma separated list of depth number. Here, `[1]` mean that this `{parte}` is called in the lemma of  the first level of `\edtext`.

I hope my explanation is clear. Could you make test and send feeback?

**Comment by maieul** [comment]
hi, some news?

**Comment by stenskjaer** [comment]
I can confirm that the solution in branch 292 works, and that is nice. 

But I must admit that the concept is a bit unclear. In this example, why should I write `\sameword[1]{parte}` when we are on the second level of `\edtext`? I think the explanation in the handbook should be clearer (and probably also with an example where it will say `[1, 2]`, because that was especially unclear). 

So, it is nice that it is possible, but I think it could be explained better, and it is a source of many potential errors if the user has to keep track of these sort of levels.

**Comment by maieul** [comment]
Yes, it is quite complex to explain.

The first thing you have to remain is that the number concerns only the case where you use \lemma, and not the case where the lemma is the first argument of the \edtext.

So in this case, it is at the first level that you use \lemma. The 1 is justified. 

Let take the problem in a other way: there is 2 \sameword in the first argument of the first edtext. However, there is only one \sameword in the \lemma associated. How could LaTeX know which one is it, except if you say to it?

If the \sameword is used in all lemma, you still can use [inlemma].

Now, some example. [] mean a \edtext level. () mean words in \lemma. 1, 2, 3 etc are the position in the list
````
[
  [1 
    [2] 
   3]
  (1�3)
  4 5
](1�5)
````

The first occurent of 1 appears in notes of level 2 and 1. So it should be marked with [1,2]. 3 appears only in notes of level 2, it should be marked by [2]. 5 appears only in notes of level 1. It should be marked by [1]. 2 appears only in notes of level 3. However, we don't use \lemma in this case, so we have to mark it (but we could).
As 1 appears in every level which use \lemma, we can use [inlemma] instead of [1,2].

Is it clear?

**Comment by stenskjaer** [comment]
Well yes, I think it is clearer to me now. Thank you for that. If you want to modify the text in the handbook, I shall happily look it through and see if I think it works and is understandable to a non-technical reader.
The structure of the different levels is nice. An example of that with real text would probably be very helpful in the handbook too.

**Comment by maieul** [comment]
ok, I will do that.


Le 11 mai 2015 � 09:32, Michael S. Christensen <notifications@github.com> a �crit :

**Comment by maieul** [comment]
please run the .dtx in the new version of the branch and make feedback on the handbook.

**Comment by stenskjaer** [comment]
I think it is good. I took the liberty to suggest som further elaborations, that I know might seem redundant. But I thinkt it is important to stress the point that only words that are referenced in the lemma-command are going to have the optional argument. 

I also added an example of the same case as above where the `edtext` is on level 2, but the `lemma` is on level 1. I think that's the most important and difficult thing to grasp. Merge or change my suggestions if you want.
